### PR TITLE
Add Background portal

### PR DESCRIPTION
**Pull request by aospiridonov**
Add Background portal

### CODE REVIEW

File: lib/src/xdg_background_portal.dart [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import 'package:dbus/dbus.dart';
+
+import 'xdg_portal_request.dart';
+
+/// Result of a request asking for an allowed to run in the background.
+class XdgBackgroundPortalRequestResult {
+  /// TRUE if the application is allowed to run in the background.
```
Comment (robert-ancell) [inline lib/src/xdg_background_portal.dart:7]: Dart uses lowercase for true (and same below).

File: lib/src/xdg_background_portal.dart [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+class XdgBackgroundPortal {
+  final DBusRemoteObject _object;
+  final String Function() _generateToken;
+
+  XdgBackgroundPortal(this._object, this._generateToken);
+
+  /// Ask to request that the application is allowed to run in the background.
+  Stream<XdgBackgroundPortalRequestResult> requestBackground({
+    String parentWindow = '',
+    String reason = '',
```
Comment (robert-ancell) [inline lib/src/xdg_background_portal.dart:40]: Make all the options arguments nullable like the other portals - sending an empty reason might be interpreted differently in the portal than not sending a reason at all.

**Comment by robert-ancell** [comment]
@aospiridonov are you resolving the conflicts?

**Comment by aospiridonov** [comment]
> @aospiridonov are you resolving the conflicts?

Done)

**Pull request merged**